example/src/main/scala/HelloWorldTwirl.scala

```diff
@@ -0,0 +1,52 @@
+import zhttp.endpoint.{*, _}
+import zhttp.http.Method.GET
+import zhttp.http._
+import zhttp.nav.Navigation
+import zhttp.service.Server
+import zio.{ExitCode, URIO}
+
+object HelloWorldTwirl extends zio.App {
+  def h2: HttpApp[Any, Nothing] = GET / "a" / *[Int] / "b" / *[Boolean] to { pathParams =>
+    val (a, b)              = pathParams.params
+    val response: UResponse = Response.text(advanced.html.index(a, b).toString())
+    response
+      .asInstanceOf[UHttpResponse]
```

Comment:
`asInstanceOf` is not required.

Reply:
Yeah that was just copied from another test.